.github/workflows/deployment.yml

```diff
@@ -101,13 +101,20 @@ jobs:
       - name: Package application image
         run: ${{ matrix.archivePortable }}
         shell: bash
-      - name: Build and publish snap
+      - name: Build snap
+        uses: jhenstridge/snapcraft-build-action@v1
+        id: snapcraft
+        if: matrix.displayName == 'linux'
+      - run: |
```

Comment:
Can you please give this also a name like `Build snap (2)`

Reply:
Done